### PR TITLE
Sdk 5

### PR DESCRIPTION
**Pull request by harshshah87**
Support for listing the commands which are in waiting state.

### CODE REVIEW

File: qds_sdk/commands.py [hunk 1]
```diff
@@ -1155,6 +1154,60 @@ def parse(cls, args):
...
+        Returns:
+            Json Unicode with command information.
+
+        """
+        conn = Qubole.agent()
+        params = {}
+        if args["status"] is not None:
+            params["status"] = args["status"]
+        else:
+            params["status"] = "done"
```
Comment (msumit) [inline qds_sdk/commands.py]: if no status is given, then it should look for all cmd statuses?

File: qds_sdk/commands.py [hunk 2]
```diff
@@ -1155,6 +1154,60 @@ def parse(cls, args):
...
+        if args["page"] is not None:
+            params["page"] = args["page"]
+        if args["per_page"] is not None:
+            params["per_page"] = args["per_page"]
```
Comment (msumit) [inline qds_sdk/commands.py]: if no `per_page` is given then how many cmds backend will send?

**Comment by msumit** [comment]
Can we also use the `ListCommand` class and `list_action` method to be available with all commands types as well? e.g. `qds.py hivecmd list --status running` will show all running hive commands only.

File: qds_sdk/commands.py (hunk 2 above)
Comment (harshshah87) [inline qds_sdk/commands.py]: per_page is 10. and page is first page by default.

File: qds_sdk/commands.py (hunk 1 above)
Comment (harshshah87) [inline qds_sdk/commands.py]: Currently, I think the API is not working as expected and asked people to look into it. But for status waiting, it works fine. So I will hold on to the PR until it gets resolved. 
But, by default, I will do an all if it is possible.

**Comment by harshshah87** [comment]
@msumit Good point. Let me check if API can work that way. If yes, I will do it.

**Pull request closed without merging**